schema/poi.go

```diff
@@ -19,7 +19,7 @@ type POI struct {
 	State           string             `bson:"state" json:"-"`
 	County          string             `bson:"county" json:"-"`
 	PlaceType       string             `bson:"place_type" json:"-"`
-	ResourceRatings POIRatingsMetric   `bson:"resource_ratings" json:"-"`
+	ResourceRatings POIRatingsMetric   `bson:"rating_metric" json:"-"`
```

Comment:
Why is the field name to be stored into database changed?

Reply:
ok

Reply:
change back to resource_ratings for consistance